app/views/home/index.html.erb

```diff
@@ -1,4 +1,54 @@
+<% if current_user && current_user.admin %>
```

Comment:
Logic in views is not recommended. You can take a look at a `policy` gem like [pundit](https://github.com/varvet/pundit) and use it in tandem with `devise` since you already have it installed. `Devise` has a `user_signed_in?` instead of testing for `current_user` and the `policy` can help you out to have different uses to the same view, for example, a visitor can see the upcoming games, but an `admin` can make changes to it.